.gitignore

```diff
@@ -7,6 +7,8 @@ node_modules
 webtau.*report*.html
 webtau-report-*.html
 *-webtau-report.html
+webtau.graphql-report.json
+run-details.actual.json
```

Comment:
this should go away when test is passes, so maybe a good idea to have it not ignored. dunno.

Reply:
you don't want to add it to git, though, right?

Reply:
50/50. Add it. No harm. Was thinking could be a good signal to remind you to rerun a test

Reply:
it is more prominent, I give you that. I'll remove it from here.